### DISCUSSION TITLE
Limit number of rows selection

### DISCUSSION THREAD

**Discussion by ifkas** [rappasoft/laravel-livewire-tables#1680] [Q&A]
Hey everyone, I wonder if anyone can enlighten me and help me, is there a way or approach to limit the total number of rows that user can select? For example maximum of 20 rows can be selected and make disabled the checkboxes or any other way of doing this?
Any input appreciated, thank you very much!

**Comment by lrljoe** [comment]
Are you talking "bulk actions" checkboxes, or per-page dropdown?

For Per-Page dropdown, the documentation [here](https://rappasoft.com/docs/laravel-livewire-tables/v3/pagination/available-methods#content-setperpageaccepted), will be useful (the "setPerPageAccepted" method).

For limiting the number of items on a page that a user can select, there's no native approach at the moment I'm afraid., you could hook into the AlpineJS "selected" object  relatively easily though.

**Comment by ifkas** [comment]
Thanks for the answer, what I want to achieve is that if user selected 20 items (checked 20 checkboxes) than to disable the rest of the checkboxes or the rows. I hope I make sense :)
